addons/pinniped/config-controller/README.md

```diff
@@ -15,4 +15,18 @@ $ ./hack/test.sh
 To run static checks (e.g., `go fmt`, `go vet`, `golangci-lint`) and the tests:
 ```sh
 $ ./hack/check.sh
-```
\ No newline at end of file
+```
+
+To generate the default pinniped addon secret:
+
+```sh 
+# manually via ytt
+ytt -f ./hack/ytt/default-package-secret.yaml -f ./hack/ytt/values.yaml -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere
+
+# using the generate script
+./hack/generate-package-secret.sh -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere 
+
+# using the make target from the top-level Tanzu Framework makefile
+# the variables here are slightly different to match what is expected in the rest of the build environment
+make generate-package-secret TKR_VERSION=v1.23.3---vmware.1-tkg.1 IAAS=vsphere
```

Comment:
are these `TKR_VERSION` and `IAAS` variables out of date now?

one thing i thought about previously and then dropped...does it make sense to document something that is in a parent tree in this README? i guess we are already using the root of t-f to install dependencies (`controller-gen`, `golangci-lint`), but seems like this line would easily get out of date if someone changes the `make` target in the t-f root `Makefile`

Reply:
Yeah, that's a good call.  So seems this is really a feature for the machinery, and humans won't touch it much.  I always have a tendency to want to doc this kind of thing, but I guess its in a comment in the root `Makefile` as well (right on the func)